app/controllers/application_controller.rb

```diff
@@ -5,6 +5,8 @@ class ApplicationController < ActionController::Base
   before_action :set_request_id
   before_action :check_maintenance_mode_is_enabled
   before_action :authenticate_and_check_access
+  before_action :set_paper_trail_whodunnit
```

Comment:
This would have caused issues in the past using basic_auth. I think now we are on more stable ground and current_user is always available?

Reply:
Yes we're using Warden for all auth providers so we always have `current_user`